package.json

```diff
@@ -3,6 +3,7 @@
   "version": "3.0.0",
   "description": "A Mocha BDD integration",
   "main": "mocha-cakes.js",
+  "typings": "mocha-cakes",
```

Comment:
This should probably be `"typings": "./mocha-cakes.d.ts"` (see http://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html)

Reply:
i fixed that to `"typings": "./mocha-cakes.d.ts"`